src/components/organisms/AssetContent/AlgorithmDatasetsListForCompute.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { ReactElement, useEffect, useState } from 'react'
+import styles from './AlgorithmDatasetsListForCompute.module.css'
+import { getAlgorithmDatasetsForCompute } from '../../../utils/aquarius'
+import AssetSelection, {
+  AssetSelectionAsset
+} from '../../molecules/FormFields/AssetSelection'
+import { useOcean } from '../../../providers/Ocean'
+import { useAsset } from '../../../providers/Asset'
+
+export default function AlgorithmDatasetsListForCompute({
+  algorithmDid
+}: {
+  algorithmDid: string
+}): ReactElement {
+  const { config } = useOcean()
+  const { type } = useAsset()
+  const [datasetsForCompute, setDatasetsForCompute] = useState<
+    AssetSelectionAsset[]
+  >()
+
+  useEffect(() => {
+    async function getDatasetsAllowedForCompute() {
+      const datasets = await getAlgorithmDatasetsForCompute(
+        algorithmDid,
+        config.metadataCacheUri
+      )
+      setDatasetsForCompute(datasets)
+    }
+    type === 'algorithm' && getDatasetsAllowedForCompute()
+  }, [type])
+
+  return (
+    <div className={styles.datasetsContainer}>
+      <span className={styles.text}>
+        Datasets algorithm is allowed to run on
```

Comment:
I think we should have the same html element and styling for this text on both compute and consume. Right now we span element here and a label in compute and have different styling for the text. See print screens 
![Screenshot 2021-05-24 at 16 59 59](https://user-images.githubusercontent.com/13447142/119359032-dabd3b80-bcb1-11eb-9c43-7347a299032d.png)
![Screenshot 2021-05-24 at 16 59 48](https://user-images.githubusercontent.com/13447142/119359046-e01a8600-bcb1-11eb-93f6-c72c7b5dc2f0.png)

Reply:
yup, that title should be also a `h2` or `h3`. Visually, after we then make that top title consistent to compute, and remove that dataset title indent this should be good:

<img width="491" alt="Screen Shot 2021-05-26 at 12 40 25" src="https://user-images.githubusercontent.com/90316/119646526-8e900980-be1f-11eb-9e20-e4d97d0731e8.png">

By visually removing the radios and external arrows I think we make it already sufficiently clear to users this list works differently.  We could enhance affordance by using some hover style, like switch row background to some slighty lighter color or switch dataset title color to `--brand-pink`

As for markup, ideally the whole `row` with that component is the `a` wrapping everything in the end so whole row is actionable, and no `label` or `input` should end up in this list.

Reply:
not sure how much refactoring we want to do here but what's bugging me a bit is that obviously we also change the meaning of the whole component with it being named `AssetSelection` and create more asset lists. From UI development perspective this sounds like we should abstract out the actual visual elements into some other component and keep it as purely visual component without functionality only switching some markup. That would apply to the whole `div.row`.

And this list here is also no form field so another reason to rather have a new component which shares that visual row component with `AssetSelection`. Just dropping this here to have it in mind cause I'm sure we don't want to over engineer here

Reply:
I agree, with the changes specified above makes more sens to make a different component because we would basically have different elements, so doesn't make sense to just switch the entire returned element but rather have a different component. I went with using the `AssetSelection` because wasn't thinking that there would be so much differences.
I can create a different component for this then.